new.html

```diff
@@ -0,0 +1,586 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>New Evaluation-MockUp</title>
+	<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+	<link rel="stylesheet" type="text/css" href="css/new.css">
+	<link rel="stylesheet" type="text/css" href="css/common.css">
+</head>
+<body>
+	<!-- Header --> 
+	<header class="header">
+		<nav class="nav">
+			<div id="logo" class="logo "></div>
+			<div id="navigation" class="navigationClass">
+				<a href="evaluations.html">
+					<div id="button1" class="button button1 formatted">
+					Evaluations
+					</div>
+				</a>
+				<div id="button2" class="button button2 formatted selected">
+					New Evaluation
+				</div>
+		</div>
+		<span class="headerSpan formatted"><a class="logOutLink" href="login.html">Logout</a></span>
+	</nav>
+</header>
+
+<!-- Main section --> 
+<section>
+	<div class="main-section">
+		<form action="" method="">
+			<div class="form-section">
+				<div class="form-section-first">
+					<input type="text" placeholder="Candidate">
+					<input type="text" placeholder="Interviewer">
+					<input type="date" class="inputDate">
+
+				</div>
+				<span class="formatted blue">Tehnical level</span></br>
+				<table>
+					<tr class="firstRow">
+						<th>Trainee</th>
+						<th>Junior</th> 
+						<th>Middle</th>
+						<th>Senior</th>
+					</tr>
+					<tr class="checkBoxes">
+						<td>
+							<input type="radio" name="level" value="trainee"><br>
+						</td>
+
+						<td>
+							<div class="radio-group">
+								<input type="radio" name="level" value="junior1"><br>
+								<input type="radio" name="level" value="junior2"><br>
+								<input type="radio" name="level" value="junior3"><br> 
+							</div> 
+						</td> 
+						<td>
+							<div class="radio-group">
+								<input type="radio" name="level" value="mid1"><br>
+								<input type="radio" name="level" value="mid2"><br>
+								<input type="radio" name="level" value="mid3"><br>  
+							</div>
+						</td>
+						<td>
+							<div class="radio-group">
+								<input type="radio" name="level" value="senior1"><br>
+								<input type="radio" name="level" value="senior2"><br>
+								<input type="radio" name="level" value="senior3"><br>  
+							</div>
+						</td>
+					</tr>
+				</table>
+			</div>
+			<div class="form-section">
```

Comment:
Semantically, this might be better marked up as an actual `section` tag. Optional, just for consideration.